gravitee-apim-console-webui/src/services-ngx/notification-settings.service.ts

```diff
@@ -33,4 +35,12 @@ export class NotificationSettingsService {
   delete(apiId: string, id: string): Observable<NotificationSettings[]> {
     return this.http.delete<NotificationSettings[]>(`${this.constants.env.baseURL}/apis/${apiId}/notificationsettings/${id}`);
   }
+
+  getNotifiers(apiId: string): Observable<Notifier[]> {
```

Comment:
What do you think about adding 2 tests for each ?

Reply:
Tests added